content/blog/2020/06/2020-06-08-hackfest-docs-results.adoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "Jenkins User Experience Hackfest Documentation Results"
+tags:
+- ux
+- documentation
+- community
+authors:
+- markewaite
+- tracymiranda
+links:
+  gitter: jenkinsci/docs
+opengraph:
+  image: /images/post-images/2020-06-08-docs-from-hackfest/docs-from-hackfest-opengraph.png
+---
+
+> Documentation is not glamorous, but is is goodness.
+> -- Thomas Otter, https://www.otteradvisory.com/2020/01/documentation-and-safety/
```

Comment:
Not blocker but wondering if the name could link to the post (just so it is more compact, but folks can still check out the source)

Reply:
Fixed in d29887f118c42f58daef152853111ac649ff4746 .  This was the first time that I'd used an ASCIIdoc quotation and I wasn't sure how far I could take it and still have the desired layout.

Looks like this now:

![image](https://user-images.githubusercontent.com/156685/84081996-4670a080-a99c-11ea-8cdf-85e9b49e88e0.png)